frontend/src/components/NotificationsBar/Notification/Notification.tsx

```diff
@@ -12,16 +12,24 @@ interface NotificationProps {
 
 const NotificationPlate: React.FC<NotificationProps> = ({ note }) => {
```

Comment:
file and component inside should have the same name. so please rename file, folder and component to Notification or NotificationPlate

Reply:
and also here you should add an index.ts file with default export in Notification folder and NotificationsBar folder

Reply:
We already have index files in these folders